### PR TITLE
create bulbs-slick-slideshow

### PR DESCRIPTION
**Pull request by brycedorn**
corresponding pr in omni: https://github.com/theonion/omni/pull/767

should I rewrite the tests that were on omni? they're mostly testing basic functionality (which can be easily manually tested), but open to it since this is a pretty heavily used feature

### CODE REVIEW

**Comment by brycedorn** [comment]
@theonion/front-end ready for 👀 (and corresponding pr in desc)

File: elements/bulbs-slick-slideshow/slick.scss [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/* Slider */
+
+.slick-slider {
+    position: relative;
+    display: block;
+    box-sizing: border-box;
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
```
Comment (spra85) [inline elements/bulbs-slick-slideshow/slick.scss]: Can we make `user-select` a mixin?

File: elements/bulbs-slick-slideshow/slick.scss [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    }
+
+    &.dragging {
+        cursor: pointer;
+        cursor: hand;
+    }
+}
+.slick-slider .slick-track,
+.slick-slider .slick-list {
+    -webkit-transform: translate3d(0, 0, 0);
```
Comment (spra85) [inline elements/bulbs-slick-slideshow/slick.scss]: We should try to use a mixin for this as well

Comment (brycedorn) [inline elements/bulbs-slick-slideshow/slick.scss]: 👍 I didn't touch this file, it's just copied from the package itself.

reasoning was (aside from webpack not playing nice with importing css from a node module) that this keeps styles from changing if the package is changed.

**Comment by brycedorn** [comment]
👍 to both comments above

File: elements/bulbs-slick-slideshow/slick.scss (hunk 2 above)
Comment (collin) [inline elements/bulbs-slick-slideshow/slick.scss]: Same here, postcss/autoprefixer should cover us here.

File: elements/bulbs-slick-slideshow/slick.scss (hunk 1 above)
Comment (collin) [inline elements/bulbs-slick-slideshow/slick.scss]: Ah yeah, I see a lot more is going on here than just prefixes. We should make a mixin and also reduce the number of prefixes if we don't need them.

File: elements/bulbs-slick-slideshow/bulbs-slick-slideshow.test.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      subject.appendChild(navElement);
+
+      attachSubject();
+      
+      sinon.spy(subject.slideshow, 'slick');
+    });
+
+    it('inits the jQuery slick carousel with the correct stuff', function () {
+      let { initialSlide } = subject;
+
```
Comment (brycedorn) [inline elements/bulbs-slick-slideshow/bulbs-slick-slideshow.test.js]: stuck on this 👇 test 🤕 

I keep getting `AssertionError: expected slick to have been called with arguments`

**Comment by spra85** [comment]
@brycedorn, take a look at the last few commits when you get in.  Fixed the test you were stuck on, will need an assist from you on testing this to make sure it still works for the slideshow content item.

**Comment by brycedorn** [comment]
@kand re-👍?

File: elements/bulbs-pollzone-map/bulbs-pollzone-map.js [hunk 4]
```diff
@@ -106,7 +106,7 @@ class BulbsPollzoneMap extends BulbsHTMLElement {
         let percent = Math.round(numVotes / totalVotes * 100);
 
         return `<li class='result'>
-          <div class='bar' style='background-color: ${this.fillKey(question.id)}; width: ${percent}%;'></div>
+          <div class='bar' style='background-color: ${this.fillKey(question['id'])}; width: ${percent}%;'></div>
```
Comment (kand) [inline elements/bulbs-pollzone-map/bulbs-pollzone-map.js]: What issue does this change resolve?

Comment (brycedorn) [inline elements/bulbs-pollzone-map/bulbs-pollzone-map.js]: this was just linting - actually removed from this PR (and merged earlier https://github.com/theonion/bulbs-elements/pull/238)

**Pull request merged**